modules/gdscript/language_server/gdscript_language_server.cpp

```diff
@@ -76,9 +79,10 @@ void GDScriptLanguageServer::thread_main(void *p_userdata) {
 }
 
 void GDScriptLanguageServer::start() {
+	host = (String)_EDITOR_GET("network/language_server/remote_host");
```

Comment:
Use the more common form of casting we use:

```suggestion
	host = String(_EDITOR_GET("network/language_server/remote_host"));
```

Reply:
Okay, no problem. Thanks!